engines/router/missionctl/log/resultlog/kafka.go

```diff
@@ -53,7 +53,7 @@ func newKafkaLogger(cfg *config.KafkaConfig) (*KafkaLogger, error) {
 }
 
 func newKafkaProducer(cfg *config.KafkaConfig) (kafkaProducer, error) {
-	producer, err := kafka.NewProducer(&kafka.ConfigMap{"bootstrap.servers": cfg.Brokers})
+	producer, err := kafka.NewProducer(&kafka.ConfigMap{"bootstrap.servers": cfg.Brokers, "message.max.bytes": 10485760})
```

Comment:
Considering this is a default value, should we make this configurable via `KafkaConfig`(i.e https://github.com/gojek/turing/blob/main/engines/router/missionctl/config/config.go#L94)?